prestoadmin/server.py

```diff
@@ -82,11 +86,202 @@ def new_sysnode_processor(node_info_rows):
 CONNECTOR_INFO_SQL = 'select catalog_name from system.metadata.catalogs'
 _LOGGER = logging.getLogger(__name__)
 
+DOWNLOAD_DIRECTORY = main_dir
+DEFAULT_RPM_NAME = 'presto-server-rpm.rpm'
+
+
+def _find_or_download_most_recent_presto_rpm():
+    latest_rpm_url = 'https://repository.sonatype.org/service/local/artifact/maven' \
+                     '/content?r=central-proxy&g=com.facebook.presto' \
+                     '&a=presto-server-rpm&e=rpm&v=RELEASE'
+    return _find_or_download_rpm(latest_rpm_url)
+
+
+def _check_if_valid_version(rpm_version):
```

Comment:
I don't think we need to check the version.  For teradata releases we'd need to have backwards compatibility for our different numbering schemes, and the whole thing could break if someone at some point decides a different versioning scheme would be better.  Also it still doesn't prevent you from specifying versions that don't exist.  I think the error message that would come up when you can't download the invalid version is just fine.